src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/CustomerThreadController.php

```diff
@@ -158,30 +189,36 @@ public function replyAction(Request $request, $customerThreadId)
      * )
      *
      * @param int $customerThreadId
-     * @param string $newStatus
+     * @param Request $request
      *
      * @return RedirectResponse
      */
-    public function updateStatusAction($customerThreadId, $newStatus)
+    public function updateStatusAction(int $customerThreadId, Request $request)
```

Comment:
```suggestion
    public function updateStatusFromViewAction(int $customerThreadId, Request $request)
```

I suggest renaming this controller to understand that two controllers are needed because they don't redirect to the same place based on the context they have been used